tests/modules/programs/opencode/custom-instructions.md

```diff
@@ -0,0 +1,2 @@
+- Always respond with emojis
+- Only use git commands when explicitly requested
```

Comment:
This doesn't appear to be used?

Reply:
True, copy paste error. I Didn't check good enough 🙈